### DISCUSSION TITLE
Multiple Bars on same screen

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [qtile/qtile#2765] [Q&A]
![Window_bar](https://user-images.githubusercontent.com/61395246/132264634-632122da-0f5c-4d87-9e8f-221e792e2500.png)
Can we get something like this in Qtile ?

Putting a widget with transparent background wont work, the compositor will apply shadow to it anyway and look bad.

**Comment by elParaguayo** [comment]
Have a look at: #2526. This should give you a way to identify the bar so you can exclude compositor rules from it.

Also, have a look at #2489.

**Reply by LamprosPitsillos** [reply]
#2489 looks amazing and can't wait to get merged!
But as for excluding the bar , that is not the wanted result, I still want picom to shadow the bar , hopefully bars.
I saw the "fake screens" in the docs, maybe that could allow for a hacky way to achieve multiple bars.

**Reply by elParaguayo** [reply]
Ah, I understand now.

Currently, the only way to get multiple bars would be to split the screen using fake screens. You can then use margins to position each bar as you want.

The problem is that your screen would be split in half and I don't _think_ you can have a window span across multiple screens with the current layouts.
